apps/studio/data/storage/bucket-object-download-mutation.ts

```diff
@@ -18,9 +18,10 @@ export const downloadBucketObject = async (
 ) => {
   if (!bucketId) throw new Error('bucketId is required')
 
-  // has to use lib/common/fetch post because the other post doesn't support wrapping blobs
+  // [Joshen] JFYI we have to use lib/common/fetch post as post from openapi-fetch doesn't support receiving octet-streams
+  // Opting to hard code /platform for non platform just for this particular mutation, so that it's clear what's happening
   const response = await post(
```

Comment:
Just for future reference we can also just opt for regular `fetch()` instead of these old methods.

Reply:
```
const headers = await constructHeaders()
const response = await fetch(`${BASE_PATH}/path`, { headers, signal })
```

Reply:
Ahh yeah good point actually, was wondering about that cause end goal i guess is to just deprecate the whole lib/common/fetch folder

Will look into that separately when i clean up the old methods! 🙏